iron-component-page.html

```diff
@@ -138,127 +138,117 @@
 
   <script>
     (function() {
-      Polymer({
-        is: 'iron-component-page',
-
-        properties: {
-          /**
-           * URL of the Polymer Analyzer descriptor to fetch and display.
-           */
-          descriptorUrl: {
-            type: String,
-            value: 'analysis.json',
-            observer: '_descriptorUrlChanged'
-          },
-
-          /**
-           * By default all routing is performed using the URL fragment
-           * (e.g. `docs.html#/elements/my-element`).
-           *
-           * If your server supports it and you would like to use the real URL
-           * path instead (e.g. `/api/docs/elements/my-element`), set this to
-           * the base path where the page is mounted, omitting the trailing
-           * slash (e.g. `/api/docs` or *empty string* for the root path).
-           */
-          baseHref: String,
-
-          /**
-           * Instead of displaying items relative to the top level of
-           * `descriptor`, start from this namespace.
-           */
-          rootNamespace: String,
-
-          /**
-           * URL prefix for demo iframes.
-           */
-          demoSrcPrefix: String,
-
-          _loading: Boolean,
-
-          _error: Object,
-
-          _descriptor: {
-            type: Object,
-            observer: '_descriptorChanged'
-          },
-
-          _path: String,
-
-          _narrow: Boolean,
-
-          _title: {
-            type: String,
-            observer: '_titleChanged'
-          }
+    Polymer({
```

Comment:
Does clang know not to indent inside the IIFE or do you have special logic?

Reply:
that's clang-format's thing

Reply:
I tested, clang does. Neat!